deebot_client/commands/advanced_mode.py

```diff
@@ -8,11 +8,19 @@ class GetAdvancedMode(GetEnableCommand):
     """Get advanced mode command."""
 
     name = "getAdvancedMode"
+
+    # TODO Potentially not available on XML based models?
+    xml_name = "GetAdvancedMode"
```

Comment:
Don't worry about on this. With the new supported_features, which I will show you next week, we don't need to care about it

Reply:
I've primarily added it for myself as a "Let me double check this", so for the moment I didn't worry to much about it :relaxed: